book-2-platform-api/chapters/VISUAL_STUDIO.md

```diff
@@ -4,19 +4,27 @@ The best way to learn a tool like Visual Studio is to use it. We'll start our ex
 
 ## Loading the Student Exercises Application in Visual Studio
 
-1. The first step is to create a new "solution" to contain our console application project. this will create a *.sln file. In the terminal, go to the folder that contains your Student Exercises application. This is the folder that has the `studentExercises.csproj` file.
+1. If you have been using Mac or Linux, the first step is to get your StudentExercise project into Winddows.
+    1. Push your changes to Github
+    1. Switch to your Windows virtual machine
+    1. Using git in Windows, pull your project from Github
+    > **Note:** The remaining steps should all be completed in Windows.
+1. Open the `Git Bash` terminal.
+1. Change directories to your Student Exercise folder. This is the folder that contains your StudentExercises.csproj.
+1. Create a new "solution" to contain our Student Exercise project.
 1. Create a new solution.
```

Comment:
probably just want one create a solution line?  or maybe the first should have been solution folder?  or maybe not.